### DISCUSSION TITLE
[Esame 13 febbraio 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#447] [Esami M-Z]
![13 feb es.2](https://github.com/user-attachments/assets/5c614402-9cd9-41aa-a409-41978a0f526b)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/8290e014-6122-480a-b7a8-f02ce5fd137a)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1024" alt="image" src="https://github.com/user-attachments/assets/2e3de2cc-9a73-47aa-a2d0-c4d4ac91d2f1" />
